crates/burn-train/src/renderer/mod.rs

```diff
@@ -1,26 +1,31 @@
+#[cfg(feature = "tui")]
+use std::io::IsTerminal;
+
 mod base;
 pub use base::*;
 
-#[cfg(not(feature = "tui"))]
 mod cli;
```

Comment:
Now `cli` may be required even if `tui` is enabled. This is because `cli::CliMetricsRenderer` is the fall-back when `stdout` is not a terminal.